staging/src/k8s.io/kubectl/pkg/util/templates/normalizers.go

```diff
@@ -70,7 +70,7 @@ type normalizer struct {
 
 func (s normalizer) markdown() normalizer {
 	bytes := []byte(s.string)
-	formatted := blackfriday.Markdown(bytes, &ASCIIRenderer{Indentation: Indentation}, blackfriday.EXTENSION_NO_INTRA_EMPHASIS)
```

Comment:
was whatever prompted use of `EXTENSION_NO_INTRA_EMPHASIS` tested, since it doesn't look like we're using it in v2

Reply:
I checked an example `./kubectl apply edit-last-applied --help` and there is no difference.

https://github.com/kubernetes/kubernetes/blob/c18c2447cf602ee1a76113200b3f5b6200a24a34/staging/src/k8s.io/kubectl/pkg/cmd/apply/apply_edit_last_applied.go#L31-L50

Some old issues before that may be related: https://github.com/spf13/cobra/issues/518. 

To keep it consistent, we can use `blackfriday.WithExtensions(blackfriday.NoIntraEmphasis)` instead. Updated.